test/test_rest_plugins.py

```diff
@@ -3707,7 +3707,8 @@
         'instance': plugins.NotifyPushover,
     }),
     # API Key + valid url_title with url
-    ('pover://%s@%s?url=my-url&url_title=title' % ('u' * 30, 'a' * 30), {
+    ('pover://%s@%s?url=my-url&url_title=title' % (
+        'u' * 30, 'a' * 30), {
```

Comment:
```suggestion
    ('pover://%s@%s?url=my-url&url_title=title' % ('u' * 30, 'a' * 30), {
```
Might as well set this back, too, to avoid the diff since no change is needed for the tests with the `map_to` change.